src/build-remote/build-remote.cc

```diff
@@ -154,7 +155,7 @@ static int main_build_remote(int argc, char * * argv)
                             }
                         }
                         if (!free) {
-                            continue;
+                            goto search;
```

Comment:
doesn't this cause a busy wait for a free builder?

Reply:
Yeah, I think now only one remote builder would get jobs.